From 18c9e5f09dfbf4d02c19c4e3dc29905507d8a760 Mon Sep 17 00:00:00 2001
From: Alon Levy <alevy@redhat.com>
Date: Mon, 3 May 2010 12:56:08 -0300
Subject: [PATCH 14/15] spice-vmc: rename guest device name to com.redhat.spice.x

RH-Author: Alon Levy <alevy@redhat.com>
Message-id: <1272891368-32706-14-git-send-email-alevy@redhat.com>
Patchwork-id: 8969
O-Subject: [RHEL-6 PATCH 13/13] spice-vmc: rename guest device name to
	com.redhat.spice.x
Bugzilla: 576488
RH-Acked-by: Arnon Gilboa <agilboa@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

From: alon.levy <alevy@redhat.com>

BZ: 576488
---
 hw/spice-vmc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/spice-vmc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hw/spice-vmc.c b/hw/spice-vmc.c
index 38ac097..a3661ae 100644
--- a/hw/spice-vmc.c
+++ b/hw/spice-vmc.c
@@ -39,7 +39,7 @@
 #include "virtio-serial.h"
 #include "qemu-spice.h"
 
-#define SPICE_VMC_GUEST_DEVICE_NAME "org.redhat.spice.0"
+#define SPICE_VMC_GUEST_DEVICE_NAME "com.redhat.spice.0"
 #define SPICE_VMC_DEVICE_NAME       "spicevmc"
 
 typedef struct {
-- 
1.7.0.3

